api/routers/metrics/metrics.py

```diff
@@ -20,20 +20,29 @@ def get_cameras(cameras: str) -> Iterator[str]:
     return [x["Id"] for x in config.values()]
 
 
+def get_all_cameras() -> Iterator[str]:
+    config = extract_config(config_type=CAMERAS)
+    return [x["Id"] for x in config.values()]
+
+
 def validate_camera_existence(camera_id: str):
     dir_path = os.path.join(os.getenv("SourceLogDirectory"), camera_id, "objects_log")
     if not os.path.exists(dir_path):
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=f"Camera with id '{camera_id}' does not exist")
 
 
 def get_areas(areas: str) -> Iterator[str]:
-    if areas and not ALL_AREAS in areas.upper():
+    if ALL_AREAS in areas.upper():
```

Comment:
Is this a list of ids? Could an id be `ALL...`? I would first split by `,` and then check if `ALL` is in there

Reply:
Yes, you are right! Thanks!